test/tests/dom/getCurrentScriptUID.js

```diff
@@ -8,6 +8,7 @@ beforeEach(() => {
     script.removeAttribute(`${ ATTRIBUTES.UID }`);
     script.removeAttribute(`${ ATTRIBUTES.UID }-auto`);
     script.setAttribute('data-csp-nonce', '654321');
```

Comment:
While you're here, should we remove `script.setAttribute('data-csp-nonce', '654321');`? data attributes are technically optional so I think we should remove it from the beforeEach. We could always add a specific test for validating the function with data attrs.